diff --git a/Application/Migrations/05_tickets.sql b/Application/Migrations/05_tickets.sql
index 5031939..24c3cbb 100644
--- a/Application/Migrations/05_tickets.sql
+++ b/Application/Migrations/05_tickets.sql
@@ -68,15 +68,14 @@ CASE
     WHEN ticket_type = 'encoding'::enum_ticket_type THEN encoding_profile_version_id IS NOT NULL
     ELSE true
 END),
-  CONSTRAINT ticket_valid_inheritence CHECK (check_fahrplan_inheritance(parent_id, fahrplan_id) IS TRUE),
-  CONSTRAINT tbl_ticket_project_fahrplanid_uq UNIQUE (project_id, fahrplan_id)
+  CONSTRAINT ticket_valid_inheritence CHECK (check_fahrplan_inheritance(parent_id, fahrplan_id) IS TRUE)
 )
 WITHOUT OIDS;
 
 -- indexes
-CREATE INDEX tbl_ticket_fahrplan_id_idx ON tbl_ticket USING btree(project_id);
+CREATE INDEX tbl_ticket_project_id_idx ON tbl_ticket USING btree(project_id);
 CREATE INDEX tbl_ticket_fahrplan_id_idx ON tbl_ticket USING btree(fahrplan_id);
-CREATE INDEX tbl_ticket_fahrplan_id_idx ON tbl_ticket USING btree(handle_id);
+CREATE INDEX tbl_ticket_handle_id_idx ON tbl_ticket USING btree(handle_id);
 
 -- trigger
 CREATE TRIGGER valid_handle BEFORE INSERT OR UPDATE ON tbl_ticket FOR EACH ROW EXECUTE PROCEDURE valid_handle();
diff --git a/Application/Migrations/20_view_parent_tickets.sql b/Application/Migrations/20_view_parent_tickets.sql
index b2d2f08..82cbca9 100644
--- a/Application/Migrations/20_view_parent_tickets.sql
+++ b/Application/Migrations/20_view_parent_tickets.sql
@@ -1,7 +1,5 @@
 ﻿BEGIN;
 
-SET search_path TO test;
-
 SET ROLE TO postgres;
 
 CREATE OR REPLACE VIEW view_parent_tickets AS
diff --git a/Application/Migrations/21_view_all_tickets.sql b/Application/Migrations/21_view_all_tickets.sql
index 968f955..e8681b6 100644
--- a/Application/Migrations/21_view_all_tickets.sql
+++ b/Application/Migrations/21_view_all_tickets.sql
@@ -1,7 +1,5 @@
 ﻿BEGIN;
 
-SET search_path TO test;
-
 SET ROLE TO postgres;
 
 CREATE OR REPLACE VIEW view_all_tickets AS 
